boskos/cmd/fake-mason/main.go

```diff
@@ -47,6 +47,7 @@ var (
 	username          = flag.String("username", "", "Username used to access the Boskos server")
 	passwordFile      = flag.String("password-file", "", "The path to password file used to access the Boskos server")
 	cleanerCount      = flag.Int("cleaner-count", defaultCleanerCount, "Number of threads running cleanup")
+	namespace         = flag.String("namespace", corev1.NamespaceDefault, "namespace to install on")
```

Comment:
This precludes a rolling upgrade, right, since we can't add the flag first, then upgrade the version, or the other way around

Reply:
The flag just got moved, it was part of the kubeOpts before